### PR TITLE
Hotfix/redirect

### PR DESCRIPTION
**Pull request by giagermain**
fixed readme description for redirects

### CODE REVIEW

**Review by sumanyuj — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -44,4 +42,4 @@ You can use the command `docker compose exec django python manage.py createsuper
 
 # Redirects
 
-Please note that requests to port 8080 are redirected to port 80 or 443 as static files are hosted through nginx. Port 80 is used in instances where self signed certificates are not present, and 443 is used when they are.
+Please note that requests to port 8080 are redirected to port 80 or 443 as static files are hosted through nginx. Port 443 is used in instances where certs are present. Port 80 is used in instances where they are not present, in which case they are then upgraded to 443 with certs.
```
Comment (WeirdAlex03) [inline README.md]: Last part is a little clunky to me
```suggestion
Please note that requests to port 8080 are redirected to port 80 or 443 as static files are hosted through nginx. Port 443 is used in instances where certs are present. Port 80 is used in instances where they are not present. Requests to port 80 will be upgraded to HTTPS (443) if that is in use.
```

**Review by WeirdAlex03 — CHANGES_REQUESTED** [review]
This will be the last PR allowed before I turn on requiring format checking to pass again

(Also for future reference, `docs/` is a more fitting branch name for this kinda thing, but I'm not gonna stop you over that)

**Review by WeirdAlex03 — APPROVED** [review]
:shipit:

**Pull request merged**